### ISSUE TITLE
Features

### ISSUE THREAD

**Issue by di48l069** [marc-ohlmann/terminal-based-chat-messenger#2]
hi, i like your terminal chat is it possible for you to add more features?

**Comment by marc-ohlmann** [comment]
Yeah definitely. Some other features I was thinking about implementing but never did include adding a /who command to see a list of the connected users, and perhaps a /tell command to send a message to a specific user. Feel free to add more features of your own if you'd like.
